From 3139f1c16011215c9c271544643b1f048f300c09 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sun, 20 Nov 2022 20:20:42 +0800
Subject: [PATCH 19/56] Revert "drm: Use size_t type for len variable in
 drm_copy_field()"

This reverts commit e7d701800365d2dd3ef2252497dcaed9a2d2fada.
---
 drivers/gpu/drm/drm_ioctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/drm_ioctl.c b/drivers/gpu/drm/drm_ioctl.c
index a15d55d06510..4606cc938b36 100644
--- a/drivers/gpu/drm/drm_ioctl.c
+++ b/drivers/gpu/drm/drm_ioctl.c
@@ -473,7 +473,7 @@ EXPORT_SYMBOL(drm_invalid_op);
  */
 static int drm_copy_field(char __user *buf, size_t *buf_len, const char *value)
 {
-	size_t len;
+	int len;
 
 	/* don't overflow userbuf */
 	len = strlen(value);
-- 
2.25.1

